pkgs/desktops/gnome-3/3.20/devtools/nemiver/bool_slot.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/dbgengine/nmv-dbg-common.h	2014-07-09 10:36:05.000000000 +0200
```

Comment:
Thanks, I prefer patches because external patches tend to be unreliable in the long term.